src/client/CodeEditor/json1PresenceDisplay.ts

```diff
@@ -44,7 +50,7 @@ export const json1PresenceDisplay = ({
         //  * Keys are presence ids
         //  * Values are presence objects as defined by ot-json1-presence
         const presenceState = {};
-
+        const  { openTab } = useContext ( VZCodeContext );
```

Comment:
Close but no cigar! `useContext` will not work here because we're not inside a React function component. You can try passing in `openTab` as an option to ` json1PresenceDisplay` instead, which should work. Good luck!